grafana/Makefile

```diff
@@ -0,0 +1,24 @@
+NAME=heapster_grafana
+VERSION=$(shell cat VERSION)
+
+dev:
+	@docker build -f Dockerfile.dev -t $(NAME):dev .
+	@docker run --rm \
+		-v $(PWD):/go/src/github.com/GoogleCloudPlatform/heapster/grafana \
+		-e INFLUXDB_SERVICE_NAME=INFLUXDB \
+		-e INFLUXDB_SERVICE_HOST=172.30.17.250 \
```

Comment:
For testing purposes, can we can spawn an influxdb container and use that instead?